lib/peep.rb

```diff
@@ -0,0 +1,35 @@
+require 'pg'
+
+class Peep 
+
+  attr_reader :peep, :time_stamp
+
+  def initialize(peep:, time_stamp:)
+    @peep = peep
+    @time_stamp = time_stamp
+  end
+
+  def self.create(peep:, time_stamp:)
+    if ENV["ENVIRONMENT"] == "test"
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+    result = connection.exec("INSERT INTO peeps(peep, time_stamp) VALUES ('#{peep}','#{time_stamp}') RETURNING peep, time_stamp;")
+    Peep.new(peep: result[0]['peep'], time_stamp: result[0]['time_stamp'])
+  end
+
+  def self.all
+    if ENV["ENVIRONMENT"] == "test"
```

Comment:
Look into moving the 'choose database' to a separate file which only has the if/else statement and then require that in the app.rb. https://github.com/makersacademy/course/blob/master/bookmark_manager/walkthroughs/14.md (about half way down)